mods/ra/maps/soviet-08a/soviet08a.lua

```diff
@@ -53,7 +53,15 @@ end
 AddParadropReinforcementTrigger = function()
 	Trigger.AfterDelay(DateTime.Seconds(90), function()
 		Media.PlaySpeechNotification(ussr, "ReinforcementsArrived")
-		scripteddrop.SendParatroopers(ScriptedParadrop.CenterPosition, false, 10)
+		scripteddrop.ActivateParatroopers(ScriptedParadrop.CenterPosition, 10)
+	end)
+end
+
+IdleHunt = function(unit)
```

Comment:
Cant see any changes using this function?

Reply:
I think this was a bug fix, but it was now too long ago for me to remember if I tested it.
The `ChurchAmbushTrigger` below calls` IdleHunt` and I guess it wasn't defined before now.